P5/Source/Guidelines/en/HD-Header.xml

```diff
@@ -2519,7 +2519,11 @@ grouping makes it easy to avoid accidentally flagging non-TEI data as errors
 during validation of the file against a TEI schema. The
 <gi>xenoData</gi> element, which may appear in the TEI header after
 the <gi>fileDesc</gi> but before the optional <gi>revisionDesc</gi>,
-is provided for this purpose.
+is provided for this purpose. Where the use of <gi>xenoData</gi> is primarily
+to store linked data, contextual information, or stand-off annotations 
```

Comment:
Perhaps
~~~xml
Where the use of <gi>xenoData</gi> is primarily
to store linked data, contextual information, or stand-off annotations 
that refer to the contents of the TEI file itself, then the
<gi>xenoData</gi> element may be placed inside the <gi>standOff</gi> 
element instead.
~~~
?

Reply:
Yes, the user of the modal verb makes that clearer, etc. Go ahead with your proposed wording.

Reply:
Done at e1673c3cf.